week-2/implement/to-pounds.js

```diff
@@ -3,3 +3,20 @@
 // Take this code and turn it into a reusable block of code.
 // Declare a function called toPounds with an appropriately named parameter.
 // Call this function a number of times to check it works for different inputs
+
+
+function money(penceString){
```

Comment:
Again, think about the name of the function here - is it clear what it will do?

Reply:
i can use penceToPounds instead of money